src/patternfly/components/Switch/switch.scss

```diff
@@ -55,25 +52,16 @@
     }
 
     &:not(:checked) {
-      ~ .pf-c-switch__label.pf-m-off {
-        color: var(--pf-c-switch__input--not-checked__label--m-off--Color);
-      }
-
-      ~ .pf-c-switch__label.pf-m-on {
-        color: var(--pf-c-switch__input--checked__label--m-on--Color);
+      ~ .pf-c-switch__label {
```

Comment:
I think you can take this CSS out of `&:not(:checked) {}` and just place it above `&:checked`?

Reply:
you're right